src/config/wayland.md

```diff
@@ -0,0 +1,106 @@
+applications built with Gtk+ will automatically choose the Wayland backend, but
+Qt applications might require the proper environment variable in GNOME. This is
+explained [below](#native-applications).
+
+### Stand-alone compositors
+
+Void Linux currently packages the following Wayland compositors:
+
+- Weston: reference compositor for Wayland
+- Sway: an i3-compatible Wayland compositor
+- Wayfire: 3D Wayland compositor
+- Cage: a Wayland kiosk
+
+### Native applications
+
+Qt5 based applications require the installation of the `qt5-wayland` package and
+setting the environment variable `QT_QPA_PLATFORM=wayland-egl` to enable their
+Wayland backend. Some KDE specific applications might also require the
+installation of the `kwayland` package. Gtk+ applications should do it
+automatically, but setting `GDK_BACKEND=wayland` can force the Wayland backend
+(however, this can break applications such as Chromium).
+
+#### Web browsers
+
+Mozilla Firefox ships with a Wayland backend which is disabled by default.
+Launching Firefox with `MOZ_ENABLE_WAYLAND=1` is required to enable it.
+
+The Midori browser, which has a Gtk+ interface, should also work on Wayland.
+
+Qt5 based browsers, such as qutebrowser, also work on Wayland.
+
+#### Terminal emulators
+
+- Alacritty (it's the standard terminal emulator for Sway)
+- Kitty
+- Sakura
+
+#### Media applications
+
+Both mpv and VLC work on Wayland, with hardware acceleration (if available and
+enabled).
+
+For image viewing, imv works natively on Wayland.
+
+#### General desktop utilities
+
+- For screenshots: `grim` and `slurp`. `slurp` is required to define a geometry.
+- For accessing the clipboard: `wl-clipboard`.
+- For displaying notifications (notification daemon): `mako`.
+- For launching applications:
+   - wofi
+   - bemenu
+   - dmenu-wayland
+
+#### Running X applications inside Wayland
+
+If an application still doesn't support Wayland, it can still be run in a
+Wayland context. XWayland is an X server that bridges this gap for most Wayland
+compositors, and is installed as a dependency for most of them. Its package is
+`xorg-server-xwayland`. For Weston, the correct package is `weston-xwayland`.
+
+## Configuration
+
+The Wayland API uses the `XDG_RUNTIME_DIR` environment variable to determine the
+directory the Wayland socket. If this environment isn't already set in your
+setup, you can add this snippet to your shell profile or to a script that
+starts-up your Wayland compositor of choice. This shouldn't be necessary with
+any of the desktop environments.
+
+```
+if test -z "${XDG_RUNTIME_DIR}"; then
+    export XDG_RUNTIME_DIR=/tmp/${UID}-runtime-dir
+    if ! test -d "${XDG_RUNTIME_DIR}"; then
+        mkdir -p "${XDG_RUNTIME_DIR}"
+        chmod 0700 "${XDG_RUNTIME_DIR}"
+    fi
+fi
+```
+
+It is also possible that some applications use the `XDG_SESSION_TYPE`
+environment variable in some way, which requires that you set it to `wayland`.
+
+## Video drivers
```

Comment:
This part is important, I wouldn't put it at the end. Having `sway` crash because you forgot the drivers is a common mistake, and not immediately recognized by everybody. Ideally, the drivers would be pulled in as a dependency as soon as you install a GL application (Wayland compositor), but this requires `mesa` to be a dependency of `libglvnd` which is not trivial at the moment.

Reply:
> this requires mesa to be a dependency of libglvnd which is not trivial at the moment.

Is there any help I can give in that front? Would it be a packaging issue?

I mostly followed the style of the Xorg page for the position of everything. I will see if I can find a better location for Drivers.

Reply:
I moved the Video Drivers to below DEs and WMs, so I could refer to them without issue.